data/core/init.lua

```diff
@@ -329,6 +321,23 @@ local function scan_project_folder(index)
   else
     core.dir_rescan_add_job(dir, ".")
   end
+  if dir.name == core.project_dir then
+    core.project_files = dir.files
```

Comment:
I realize that this was already present, but I would remove this, considering that it's kind of useless when there are multiple project dirs. The only thing that uses `core.project_files` is this: https://github.com/lite-xl/lite-xl/blob/7b82d787c0e8a8ccd17410763e9a18958f2c5e1c/data/core/statusview.lua#L142

and it should use `core.project_files_number()`.

Reply:
Aha. Yeah. Good call. I'll change it.